Loop Status Extension/StatusViewController.swift

```diff
@@ -205,11 +205,24 @@ class StatusViewController: UIViewController, NCWidgetProviding {
                 if let eventualGlucose = predictedGlucose.last {
                     let formatter = NumberFormatter.glucoseFormatter(for: eventualGlucose.unit)
 
+                    var activeInsulinString = "--"
+                    if let activeInsulin = context.activeInsulin {
+                        // If the returned value doesn't indicate an error,
+                        // assign it to the label:
+                        if activeInsulin.error == "" {
+                            activeInsulinString = String(format: "%.1f", activeInsulin.value)
+                        } else {
+                            // This will be ugly but maybe (?) informative: 
+                            activeInsulinString = activeInsulin.error
```

Comment:
what will this look like? I haven't looked up what doseStore errors look like, but we don't want it to be misleading (ie, a small integer value here could be seriously confusing or potentially very harmful). I think I'd want to see us put in a non-numeric value of known size, like "??" or "ERR"

Reply:
This should use NumberFormatter, not "%.1f "

Reply:
@ps2  Happy to switch to NumberFormatter, but for my own education, what's the advantage there?  Is it that it handles locale information?